sky/core.py

```diff
@@ -488,6 +488,19 @@ def autostop(
                 f'  {_stop_not_supported_message(handle.launched_resources)}.'
             ) from e
 
+    # Check if autodown is required and supported
+    if down and not is_cancel:
```

Comment:
We can check this with `not is_cancel` only, since `autostop` and `autodown` should have similar requirements for the cloud support.